packages/core/src/types.ts

```diff
@@ -1635,6 +1641,8 @@ export interface InterpreterOptions {
   autoForward?: boolean;
 
   sync?: boolean;
+
+  state?: any; // TODO: type this
```

Comment:
this should be `TPersisted`, right? 🤔

Reply:
739d8cbc8920ef5f2a76b16316a82b059a53c1da